cookbooks/fb_apcupsd/templates/default/apcupsd.conf.erb

```diff
@@ -0,0 +1,8 @@
+## apcupsd.conf v1.1 ##
+#
+# This file is maintained by Chef. Do not edit, all changes will be
+# overwritten. See fb_apcupsd/README.md
+
+<% node['fb_apcupsd']['config'].to_hash.each do |key, val| -%>
```

Comment:
does it make sense to sort these ? with an API like this the file can change order drastically as different people call into it.